pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>de.evoila.kusim</groupId>
+    <artifactId>helm-wrapper</artifactId>
+    <version>0.1.3-SNAPSHOT</version>
+    <description>KuSIM Helm Wrapper</description>
+
+    <properties>
+        <registry.imagePath>harbor.evoila-lab.work/bu-dev/kusim/images</registry.imagePath>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <extensions>true</extensions>
```

Comment:
is this line necessary? documentation sounds like it is needed for custom lifecycles

Reply:
I will check again, from the examples I checked it was there every time (even for simple ones like this)